addons/mail/static/src/models/thread/thread.js

```diff
@@ -610,6 +610,24 @@ function factory(dependencies) {
                     followers: [['unlink-all']],
                 });
             }
+            this.getSuggestedPartners();
+        }
+
+        async getSuggestedPartners() {
+            const result = await this.async(() => this.env.services.rpc({
+                route: '/mail/get_suggested_recipients',
+                params: {
+                    model: this.model,
+                    res_ids: [this.id],
+                },
+            }));
+            if (result[this.id]) {
+                this.update({
+                    suggestedPartners: [['insert-and-replace', result[this.id].map(data => {
+                        return this.env.models['mail.suggestedPartner'].convertData(data);
+                    })]],
+                });
+            }
```

Comment:
alphabetical order (keyword `async` doesn't count), and that's not really a "get" method, more like a "fetch (and update)" one.